core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -690,26 +845,39 @@ void core_mmu_get_entry_primitive(const void *table, size_t level,
 
 bool core_mmu_user_va_range_is_defined(void)
 {
-	return user_va_idx != -1;
+	bool ret;
+	uint32_t exceptions = thread_mask_exceptions(THREAD_EXCP_ALL);
```

Comment:
For this thread the partition is supposed to remain constant so masking of exceptions shouldn't be needed.